open-company/open-company-web
# User dropdown menu restyle and improvement.

Card: https://trello.com/c/LaHWYKSn

Abstract: https://share.goabstract.com/bce92c7a-6c96-4c11-96f8-5753b89be0c4

To test:
- enter as Admin
- open the user menu
- [x] do you see all the needed menu items? Good
- [x] all the items work? Good
- enter as Author
- open the user menu
- [x] do you see all the needed menu items? Good
- [x] all the items work? Good
- enter as Viewer
- open the user menu
- [x] do you see all the needed menu items? Good
- [x] all the items work? Good
